Vagrantfile

```diff
@@ -117,7 +117,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       ansible.raw_arguments = ["--timeout=60"]
     end
 
-    worker.vm.provision "shell", inline: "service celeryd restart >> /dev/null 2>&1", run: "always"
+    worker.vm.provision "shell" do |shell|
```

Comment:
I believe that this is now missing the `run: "always"` option. Not entirely sure this provisioner is still necessary, but I think it would be safest to keep things as they were since this change doesn't seem related to the PR.